spec/file_list_spec.rb

```diff
@@ -0,0 +1,17 @@
+require_relative '../lib/forspell/file_list'
+
+RSpec.describe Forspell::FileList do
+  describe 'including path' do
+    subject { described_class.new(paths: paths, exclude_paths: exclude_paths) }
+
+    describe 'correct behavior' do
+      let(:paths) { ['lib'] }
+      let(:exclude_paths) { ['spec'] }
+      it 'should contain only files in lib' do
+        subject.each do |file| 
+          expect(file.start_with? 'lib').to be_truthy
+        end
+      end
+    end
+  end
+end
```

Comment:
I believe it should be several tests for edge cases like we described: if there is files or folders (and if path is `README.md Contributing.md` and exclude is `Contributing.md`), if there are some non-existent pathes, what should it do -- raise or notify or what; and what if `exclude` excludes all of the `include` :)